modules/ROOT/pages/rtm-agent-whitelists.adoc

```diff
@@ -71,6 +77,28 @@ In addition, the following endpoints use WebSockets:
 |Region |Name |IP Address
 |*US*|runtime-manager.anypoint.mulesoft.com |18.214.68.14
 |*US*|runtime-manager.anypoint.mulesoft.com |35.174.151.175
+|*US-GOV*|runtime-manager.gov.anypoint.mulesoft.com |52.61.85.119
+|*US-GOV*|runtime-manager.gov.anypoint.mulesoft.com |3.32.47.1
+|*US-GOV*|runtime-manager.gov.anypoint.mulesoft.com |160.1.154.225
+|===
+
+* Whitelist these static IPs in the `US` region to access the `data-authenticator` hosts:
++
+[%header,cols="10a,60a,30a"]
+|===
+|Region |Name |IP Address
+|*US-GOV*|data-authenticator.gov.anypoint.mulesoft.com |15.200.57.78
+|*US-GOV*|data-authenticator.gov.anypoint.mulesoft.com |15.200.174.74
+|*US-GOV*|data-authenticator.gov.anypoint.mulesoft.com |160.1.51.12
+|===
+
+* Whitelist these static IPs in the `US` region to access the MuleSoft Government Cloud hosts:
```

Comment:
same comment as above. Since we are moving away from the use of terms such as "Blacklist" and "Whitelist", maybe in such cases where we mean to say allow or block, and the term is not coming from a UI text, maybe we can start making the change.